app/services/candidates/registrations/registration_step.rb

```diff
@@ -0,0 +1,22 @@
+module Candidates
+  module Registrations
+    class RegistrationStep
+      include ActiveModel::Model
+      include ActiveModel::Attributes
+
+      attribute :created_at, :datetime
+      attribute :updated_at, :datetime
+
+      def persisted?
+        created_at.present?
+      end
+
+      # Flag the model as having being persisted
+      def persisted!
```

Comment:
`persisted` isn't a action? its a status?

should this be `persist!` or `save!` ?

Reply:
I've gone for  changing it to `flag_as_persisted!` as this method doesn't really save the record, it's called by the "saver" `RegistrationSession#save(model)` in a future PR. (Also has the added benefit of removing the clarifying comment!)